starport/pkg/xgenny/run.go

```diff
@@ -2,12 +2,25 @@ package xgenny
 
 import (
 	"context"
+	"errors"
+	"os"
 
 	"github.com/gobuffalo/genny"
 	"github.com/gobuffalo/logger"
 	"github.com/tendermint/starport/starport/pkg/placeholder"
+	"github.com/tendermint/starport/starport/pkg/validation"
 )
 
+var _ validation.Error = (*dryRunError)(nil)
+
+type dryRunError struct {
```

Comment:
It would be better to make this public since and instance of this returned by a public API.

Reply:
it can be recovered as validation.Error